components/Init/src/main.c

```diff
@@ -615,6 +598,16 @@ void make_virtio_net_vswitch(vm_t *vm)
     return make_virtio_net_vswitch_driver(vm, pci, io_ports);
 }
 
+void make_virtio_con(vm_t *vm)
+{
+    return make_virtio_con_driver(vm, pci, io_ports);
```

Comment:
Why is there this indirection here?

Reply:
All the make_virtio_X functions exist in main.c currently. They could be defined within their own respective files, however.

Reply:
Why do we have both `make_virtio_con()` and `make_virtio_con_driver()`?
Why not just rename `make_virtio_con_driver()` as `make_virtio_con()` ????

Reply:
I believe the original intent was to have all CAmkES interacting functions within `camkes-vm`, so all functions called by CAmkES are defined and implemented within `camkes-vm`.